pkg/gui/controllers/local_commits_controller.go

```diff
@@ -1262,6 +1262,11 @@ func (self *LocalCommitsController) handleOpenLogMenu() error {
 								OnPress: onPress("topo-order"),
 								Widget:  types.MakeMenuRadioButton(currentValue == "topo-order"),
 							},
+							{
+								Label:   "default",
+								OnPress: onPress("default"),
+								Widget:  types.MakeMenuRadioButton(currentValue == "default"),
+							},
```

Comment:
Any particular reason why you put it at the second position in the menu? I would have expected it to go last, as it's probably the least common setting to use. As far as I understand, the only reason to choose it really is when you have a huge repository and want to speed things up a bit.

Reply:
You are correct, moved "default" to last.